### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by FlorianHase**
releasing a preliminary version of chimera

### CODE REVIEW

**Review by tassoneroberto — APPROVED** [review]

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+numpy
```
Comment (Sebastiencreoff) [inline requirements.txt:1]: I think, we do not need it as it is not present inside setup.pu

**Review by lroch — APPROVED** [review]

File: src/chimera/__init__.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env python 
+
+#==============================================================================
+
+__version__ = '0.1.0'
```
Comment (Sebastiencreoff) [inline src/chimera/__init__.py:5]: maybe set in another file. `__about_.py` and include inside setup.py to use it.
`about = {}
version_file = pathlib.Path('.') / 'src' / 'chimera' / '__about__.py'
exec(version_file.read_text(), about)
`

and then about['version']

File: tox.ini [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    pytest
+    pytest-cov>=2.8.1
+    pytest-randomly
+
+commands =
+    pip install -r requirements.txt
+
+[testenv:py{36,37}]
+commands =
+    pip install -r requirements.txt
```
Comment (Sebastiencreoff) [inline tox.ini:16]: not needed if you delete numpy

File: setup.py [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+        "Intended Audience :: Science/Research",
+        "Operating System :: Windows/Unix",
+        "Programming Language :: Python",
+        "Topic :: Scientific/Engineering",
+    ],
+    author="ChemOS Inc.",
+    author_email="florian@chemos.io",
+    package_dir={"chimera": "src/chimera"},
+    packages=find_packages("src"),
+    zip_safe=False,
```
Comment (Sebastiencreoff) [inline setup.py:31]: add as dependancy numpy.

File: requirements.txt (hunk 1 above)
Comment (FlorianHase) [reply requirements.txt:1]: We do need numpy. Added it to setup.py

File: src/chimera/__init__.py (hunk 2 above)
Comment (FlorianHase) [reply src/chimera/__init__.py:5]: Ok, updated. I believe that this way of setting the version is the solution that you suggested earlier for the phoenics repo. Which strategy should we pick going forward?

**Pull request merged**